internal/config/broadcaster.go

```diff
@@ -71,7 +73,10 @@ func (b *broadcasterer) Broadcaster() Broadcaster {
 			panic(fmt.Errorf("broadcaster: invalid airdrop amount: %w", err))
 		}
 
-		cosmosRPC, err := grpc.Dial(cfg.CosmosRPC, grpc.WithInsecure(), grpc.WithKeepaliveParams(keepalive.ClientParameters{
+		tlsConfig := &tls.Config{
+			InsecureSkipVerify: true,
+		}
+		cosmosRPC, err := grpc.Dial(cfg.CosmosRPC, grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)), grpc.WithKeepaliveParams(keepalive.ClientParameters{
```

Comment:
Do you really need this tls things, will `grpc.WithTransportCredentials(insecure.NewCredentials())` be okay for you?

Reply:
Yes I need, otherwise I get: "error reading server preface: http2: frame too large". It's a magic...)

Reply:
Really, we use this approach in ldif-sdk and it works, but okay, let it be so